testsuite/features/step_definitions/common_steps.rb

```diff
@@ -333,7 +333,7 @@
   $server.run_until_ok("ls /var/cache/rhn/repodata/#{channel}/updateinfo.xml.gz")
 end
 
-When(/^I push package "([^"]*)" into "([^"]*)" channel$/) do |arg1, arg2|
+And(/^I push package "([^"]*)" into "([^"]*)" channel$/) do |arg1, arg2|
```

Comment:
Actually is better to keep When here, you can always use `And` in your scenario if you need. But keep `When` here for clarification.